python/lsst/daf/butler/cli/utils.py

```diff
@@ -196,6 +196,9 @@ def addArgumentHelp(doc, helpText):
         # (Lines after the first blank line will be argument help.)
         while len(doclines) > 1 and doclines[1]:
             doclines[0] = " ".join((doclines[0], doclines.pop(1).strip()))
+        # Add standard indent to help text for proper alignment with command
+        # function documentation:
+        helpText = "    " + helpText
```

Comment:
Word wrapping in click does seem harder than it should be...

Reply:
This is partially my fault, for modifying how click.argument help is inserted; click says you should add it yourself in the command function doctoring, but I wanted to reuse the help text along with the argument declaration via the argument decorator. I'm kind of fighting the API. If it's a continuous PITA then probably I should back out what I've done & let the wookie win.